src/main/java/com/gooddata/executeafm/resultspec/Dimension.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) 2007-2017, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.executeafm.resultspec;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.gooddata.util.GoodDataToStringBuilder;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+import static com.gooddata.util.Validate.notEmpty;
+import static com.gooddata.util.Validate.notNull;
+import static java.util.Arrays.asList;
+
+/**
+ * Dimension content definition. Dimension contains one or more attributes.
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class Dimension {
+
+    /**
+     * Marker element to be used within itemIdentifiers marking the placement of measures
+     */
+    public static final String MEASURE_GROUP = "measureGroup";
+
+    private final String name;
+    private final List<String> itemIdentifiers;
```

Comment:
Isn't this "item" always attribute?

Reply:
no it's not, it can be attribute, measure or measureGroup

Reply:
described in javadoc